src/components/File.jsx

```diff
@@ -143,6 +145,7 @@ class File extends Component {
     const { filename, extension } = splitFilename(attributes)
     return (
       <div className={classes}>
+        { attributes.links && <Preview thumbnail={`${cozy.client._url}${attributes.links.small}`} /> }
```

Comment:
I understand we need the server's url here, but I think we should add a abstraction layer to avoid the strong coupling between the external lib and the component. May the thumbnail's url be loaded via a Props to the `<File />` dumb component?